packages/docs/docs/api/javascript/defineConsts.mdx

```diff
@@ -10,7 +10,7 @@ sidebar_position: 3
 
 Defines static style constants that can be used directly in `create` calls anywhere in the codebase. Unlike `defineVars`, these values are inlined at build time and do not generate actual CSS variables.
 
-*Note: `defineConsts` is an experimental feature and cannot currently be used with `firstThatWorks` or when `runtimeInjection` is enabled.*
+*Note: `defineConsts` is an experimental feature and cannot currently be used with `firstThatWorks`, dynamic styles, or when `runtimeInjection` is enabled.*
```

Comment:
1. `firstTheWorks` will never be supported. It's not really supported for variables in general and you're supposed to use `@supports` to check what is valid instead.

2. dynamic Styles should work very easily if we just export object value given to `defineConsts` as is in the generated JS.